### DISCUSSION TITLE
[20240519] Full screen UI freezes with iPadOS

### DISCUSSION THREAD

**Discussion by kw-sudo** [rern/rAudio#1674] [Fixed Issues]
Hi rern,

I got a new SD card, did a fresh install (i20240480 > 20240519) and put a link to the full UI on the home screen.

As soon as the lock screen is activated or I close the screen cover and call up the full UI again, the interface has no function. The same happens in browser mode. I have to close the task and call it up again, then it works once.

- iPad Air 2 (iPadOS 15.8.4) and 3 (iPadOS 17.5)
- RasPi 3 v1.2

PS: for a quick test, I still have an old installation based on 20230101, this works fine.

Regards,
Oliver

**Comment by kw-sudo** [comment]
New insights...

PS: due to a personal 'hooked on project' (tidal-dl), I had to do _pacman -Syu_ (for python/pip3). With that update all previous rAudio versions failed afterwards.

PPS: the previously mentioned version above has already a working pip3-instance inside, so there's no nead to -syu

**Comment by cYCL157** [comment]
I have had this issue for quite awhile, sometime earlier this year I think, I have posted a few bugs on it but can't track down what I think it specifically is -- but I think the websocket is disconnected and does not reconnect correctly.  When you get to that state in the UI, can you do anything at all or just not actions that would impact playing music (stop, play, skip, volume, etc)?

**Comment by kw-sudo** [comment]
Only the 'playing music' actions, no matter whether simple cover mode nor buttons. But I can swipe left/right, so I'm able to 'kill' the playlist.

**Comment by ha11oga11o** [comment]
Sorry for hijack, but could this problem cause one im trying to overcome?

#1661
